AmplifyPlugins/Auth/Sources/AWSCognitoAuthPlugin/Task/AWSAuthSignUpTask.swift

```diff
@@ -34,6 +35,16 @@ class AWSAuthSignUpTask: AuthSignUpTask {
             throw error
         }
     }
+
+    private func didConfigure() async {
+        await withCheckedContinuation { [weak self] (continuation: CheckedContinuation<Void, Never>) in
+            stateMachineToken = authStateMachine.listen({ [weak self] state in
+                guard let self = self, case .configured = state else { return }
```

Comment:
this return will not allow the `continuation` to fail or resume and likely cause a runtime exception during development